chain/stmgr/utils.go

```diff
@@ -341,16 +341,20 @@ func LoadSectorsFromSet(ctx context.Context, bs blockstore.Blockstore, ssc cid.C
 		return nil, err
 	}
 
+	filterSize, err := filter.Count()
+	if err != nil {
+		return nil, err
+	}
+
+	filterSet, err := filter.AllMap(filterSize)
```

Comment:
If `filterSize` can be large, this is a problem. Ideally we'd iterate over each run in the bitfield while walking the sector set, but that would require some changes to the bitfield library.

Is that worth it?

Reply:
If the change to go-bitfield is just the ability to get a RunIterator from a BitFiled instance, I feel like that could be worth it

Reply:
Yeah, that's what the change would be.